docs/guides/auth.md

```diff
@@ -118,6 +118,19 @@ git:
 ```
 {% endhint %}
 
+#### Debugging tips
+
+You can run earthly with `--verbose`, which will provide debugging messages to help understand how a remote earthly reference is transformed into a git URL for cloning.
+
+You can additionally enable low-level git debugging in buildkit, by adding the following to your `~/.earthly/config.yml`:
+
+```yaml
+global:
+  buildkit_additional_args: [ '-e', 'BUILDKIT_DEBUG_GIT=1' ]
+```
+
+The buildkit logs can be displayed with `docker logs earthly-buildkitd`.
```

Comment:
This change seems like it belongs to another PR? Is this intentional?

Reply:
I think I scooped it from main by accident. But I don't see why this wouldn't be in the 0.7 docs. @alexcb is this okay with you?

Reply:
I also tried bringing 0.7-docs up to date with main. I think I was used to the `next` branch. Apologies if I shouldn't have done that

Reply:
> I also tried bringing 0.7-docs up to date with main. I think I was used to the next branch. Apologies if I shouldn't have done that

We only do that on release. `main` might contain docs for unreleased features.

Reply:
That makes sense. Sorry, I did that a bit impulsively. It looks like this is the only doc change from `main` that wasn't in `docs-0.7` yet, and doesn't correspond to an unreleased feature. 

Is the process for adding to these docs documented anywhere yet? Perhaps it can go in `CONTRIBUTING.md`. Happy to add that.

Reply:
Already there: https://github.com/earthly/earthly/blob/main/CONTRIBUTING.md#documentation ;-)

Reply:
Heh I skimmed, but missed it. Should've used ctrl-f :)

Reply:
> I think I scooped it from main by accident. But I don't see why this wouldn't be in the 0.7 docs. @alexcb is this okay with you?

Seems like you just need to change this PR to be against `docs-0.7`; currently you're PR is attempting to merge `docs-0.7` (plus additional commits) into `main` -- if this is merged as-is (please dont!), github will squash commits together, which will then lead to a conflict when we merge `main` back into `docs-0.7`.